src/Chronos.php

```diff
@@ -134,7 +134,7 @@ public function toMutable(): MutableDateTime
      *
      * @return $this
      */
-    public function copy()
+    public function copy(): ChronosInterface
```

Comment:
Same thing as in core: We should not add the typehints for chainable, it breaks usability.
Instead use the Spryker sniffs to ensure this is all clean and safe, and that solves it properly.

Reply:
Like I said on core PR, show me how it breaks usability.

That said the docblock of `ChronosInterface::copy()` has `@return static` and since chronos is supposed to be immutable shouldn't this method return a `clone $this` instead of `$this`?

Reply:
I modified `Chronos::copy()` to return cloned instance.